UI/CensusBrowseWnd.cpp

```diff
@@ -22,7 +22,10 @@ namespace {
     const int       EDGE_PAD(3);
 
     const GG::Y     ICON_BROWSE_ICON_HEIGHT(64);
-    const GG::X     BROWSE_TEXT_WIDTH(200);
+
+    const GG::X BrowseTextWidth() {
+        return GG::X (200.0f * (std::max(static_cast<float>(ClientUI::Pts()), 12.0f) / 12.0f));
```

Comment:
Agree, but did not think of a more simplified method apart from adjusting BROWSE_TEXT_WIDTH to a larger value.   As that uses more screen real estate and would account for even larger font sizes, went with a percentage adjustment.